src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -188,6 +197,10 @@ protected function commandsNamed(array $names)
      */
     protected function addImpliedCommands(Grammar $grammar)
     {
+        if ($this->hasAutoIncrementColumn() && ! $this->creating()) {
+            array_unshift($this->commands, $this->createCommand('autoIncrementStartingValues'));
+        }
+
```

Comment:
Please explain this change.

Reply:
It's to handle auto-increment starting value using `from` modifier, before this PR `from` was only supported when adding a column and creating a table. I removed this from `compileAdd` and add it here to handle both add and change. now it is supported to use `from` modifier with `change`.

Reply:
Actually it's much cleaner to handle auto-increment starting value as a fluent command for MySQL and PostgreSQL. but I didn't want to change a lot of code at once and make it hard to review.